superset-frontend/packages/superset-ui-chart-controls/src/utils/D3Formatting.ts

```diff
@@ -32,6 +32,10 @@ export const D3_NUMBER_FORMAT_DESCRIPTION_PERCENTAGE_TEXT = t(
 
 // input choices & options
 export const D3_FORMAT_OPTIONS: [string, string][] = [
+  [NumberFormats.INDIAN_FORMAT_NUMBER, t('Indian format')],
+  [NumberFormats.INDIAN_FORMAT_CURRENCY, t('Indian currency format')],
+  [NumberFormats.INDIAN_FORMAT_NUMBER_SHORT, t('Indian format (in units)')],
+  [NumberFormats.INDIAN_FORMAT_CURRENCY_SHORT, t('Indian currency format (in units)')],
```

Comment:
@mittalsuraj18 any suggestion for the placeholders?

Reply:
Looks good lol :-p

Reply:
+1